skylarktest/bazeltest.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 The Bazel Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Ugly override hack, which redefines DataFile if we assume to run inside
+// of Bazel
+
+package skylarktest
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func init() {
+	testDir := os.Getenv("TEST_SRCDIR")
+	if len(testDir) == 0 {
+		return
+	}
+
+	// Seems like we are running inside of Bazel
```

Comment:
No, Bazel or Blaze. That's why I suggest you tag this file with //+bazel, and set that tag in the Bazel go_library rule, as Blaze does not want this behavior.

Reply:
> That's why I suggest you tag this file with //+bazel, and set that tag in the Bazel go_library rule, as Blaze does not want this behavior.

Sounds reasonable. You did lose me there however. It seems like you rely on some Golang (tag) magic. Never seen this before and my Google foo does not bring up anything useful. Can you please post a link to some documentation (hoping that for once there is a documentation for a Golang feature).

Reply:
> That's why I suggest you tag this file with //+bazel

Ah that is not a Golang but rather `go build` mechanics. Added that.

>  and set that tag in the Bazel go_library 

AFAIK Bazel does not care about `go build` shenanigans so no tags to add. Works just fine ass is.